Assets/StreamingAssets/Localization/en_US.lang

```diff
@@ -1,3 +1,4 @@
+en_US=English (US)
```

Comment:
IMO I think that repeating the language code as the key is unnecessary. inside each lang file there should instead be  `lang=English (US)`. that way the keys are consistent across all lang files. in your approach only english will have `en_US` only hebrew will have `he_IL` causing inconsistencies about which keys should exist in which files